astroid/tests/unittest_brain.py

```diff
@@ -1697,7 +1697,7 @@ def test_infer_dict_from_keys():
         assert sorted(actual_values) == ["a", "b", "c"]
 
 
-class TestFunctoolsPartial:
+class TestFunctoolsPartial(unittest.TestCase):
```

Comment:
@hippo91 It's not required to add `unittest.TestCase` here because we use `pytest` and it is able to find these tests accordingly.